src/uct/ib/mlx5/ib_mlx5.inl

```diff
@@ -504,16 +504,20 @@ size_t uct_ib_mlx5_set_data_seg_iov(uct_ib_mlx5_txwq_t *txwq,
 static UCS_F_ALWAYS_INLINE void uct_ib_mlx5_bf_copy_bb(void * restrict dst,
                                                        void * restrict src)
 {
-#if defined( __SSE4_2__)
-    UCS_WORD_COPY(__m128i, dst, __m128i, src, MLX5_SEND_WQE_BB);
-#elif defined(__ARM_NEON)
+#if defined(__ARM_NEON)
     UCS_WORD_COPY(int16x8_t, dst, int16x8_t, src, MLX5_SEND_WQE_BB);
```

Comment:
`vst4q_u64(dst, vld4q_u64(src));` ?

Reply:
if we need it in v1.16.x i'd rather backport it as a separate commit, @yosefe I backport that one? it was not coming from a bug report unless i'm mistaken.

Reply:
I see... that was in a different PR. Yes, agree that this doesn't belong to this PR.